help/en/package/jmri/jmrit/logixng/InlineLogixNG.shtml

```diff
@@ -65,6 +65,25 @@
       <p>If <strong>Yes</strong> is selected, the LogixNG will be deleted and the object remove
       process will continue.  If <strong>No</strong> is selected, the object remove will not occur.</p>
 
+      <h2 id="logixng_icon">LogixNG Icon<span class="since">since 5.5.5</span></h2>
+
+      <p class="noted">This feature is currently limited to Panel Editor panels.  For use with
+      Control Panel Editor panels, switch the view to Panel Editor, add the icon and then switch
+      back to Control Panel Editor.</p>
+
+      <p>The <strong>LogixNG Icon</strong> extends the Inline LogixNG feature to <strong>invoke</strong>
+      the attached Inline LogixNG when clicked.  The ConditionalNGs within the attached LogixNG will
+      be executed in sequence.</p>
```

Comment:
A suggestion is to add the line:
`To prevent that the ConditionalNG is executed on startup, unselect the checkbox "Startup" for the ConditionalNG.`

I suppose that the LogixNG Icon will be uses for cases there the user for example wants to run a script or set a couple of sensors. And the user probably doesn't want to do that when the panel is loaded.